jbmc/regression/jbmc/assume-inputs-non-null/My.java

```diff
@@ -0,0 +1,41 @@
+class My {
+
+  String field;
+
+  public static void stringArrayArg(String[] arg) {
+    if (arg == null) {
+      assert false;
```

Comment:
:bulb: It might make the tests a little easier to comprehend if you _only_ put the `assert false` in the branches that shouldn't be covered. 
Indeed could even be written as non-coverage style:

```java
assert(arg != null)
```

Then you only need to check for verification successful.

Reply:
I have a slight preference for the branching solution (using `if`s) as I would like to keep both assertions (to be sure that a trace exists at all), and I don't want to make too many assumptions as to whether the `assert` acts as a guard or not.